dashboard/components/RelationDependencyGraph.tsx

```diff
@@ -236,3 +277,44 @@ export default function RelationDependencyGraph({
     </>
   )
 }
+
+/**
+ * The color for the edge with given back pressure value.
+ *
+ * @param value The back pressure rate, between 0 and 100.
+ */
+function backPressureColor(value: number) {
```

Comment:
nit: reuse code in `FragmentGraph`?

Reply:
Done: https://github.com/risingwavelabs/risingwave/pull/18280/commits/3624e3aa954c0f2156aedb842898042579df8535